src/main.js

```diff
@@ -218,6 +218,13 @@ allowProtocolDialog.init(mainWindow);
 // This method will be called when Electron has finished
 // initialization and is ready to create browser windows.
 app.on('ready', function() {
+  ipcMain.on('notified', function(event, arg) {
```

Comment:
https://nodejs.org/api/events.html#events_emitter_on_eventname_listener You CAN add multiple event listeners to an `EventEmitter`, which ipcMain is :-)